checker/manual/troubleshooting.tex

```diff
@@ -741,9 +736,32 @@ \subsection{Build the Checker Framework\label{building}}
 \end{Verbatim}
 % $ to unconfuse Emacs LaTeX mode
 
+Once \<checker.jar> is built, you may wish to put the Checker Framework \<javac> even earlier on
+your path. The Checker Framework's \code{javac} ensures that \<checker.jar> 
+is on your classpath and bootclasspath, but is otherwise
+identical to the Type Annotations compiler:
+
+\begin{Verbatim}
+export PATH=$JSR308/checker-framework/checker/bin:$JSR308/jsr308-langtools/dist/bin:${PATH}
+\end{Verbatim}
+
+If you are developing Checkers and wish to use the latest builds of your source files compiled by
```

Comment:
It's clearer to use singular "a Checker" versus multiple "Checkers", since this applies to someone who is just creating one.

Also, I don't think that "Checker" needs to be capitalized here.